scripts/npm/prepublish/index.js

```diff
@@ -157,12 +158,12 @@ function compareVersionNumber( component ) {
       util.printLn.info( component + ' remains ' + npmVersion, true );
     }
   } ).catch( function( err ) {
-    if ( /returned 404/.test( err ) ) {
+    if ( ( /returned 404/ ).test( err ) ) {
       util.printLn.success( component + ': ' + localVersion, true );
       return {
         name:       component,
         newVersion: localVersion,
-        oldVersion: undefined
+        oldVersion: UNDEFINED
```

Comment:
Rather than specifically setting undefined, can we either leave off the key or set it to null?

Reply:
@contolini What do you think this should be? I'll merge this PR, but we can do another PR for this.